pkg/chunk/table_manager.go

```diff
@@ -166,6 +167,10 @@ func (m *TableManager) Stop() {
 func (m *TableManager) loop() {
 	defer m.wait.Done()
 
+	// Sleep for a bit to spread the sync load across different times if the tablemanagers are all started at once.
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I think this is unwise anywhere outside of a main program.  Sometimes you want repeatable random numbers, e.g. to reproduce a bug.

Also if `loop()` were to be called twice in the same program at the same time (within clock resolution) you'd get the opposite of what you intended.

Reply:
I've moved this one level up. Let me know if that makes more sense?

Reply:
You're still calling `rand.Seed()` in a place that is not the `main()`, and I still think that is unwise.